Signum.Entities.Extensions/Workflow/WorkflowEvent.cs

```diff
@@ -47,11 +54,24 @@ public void SetModel(ModelEntity model)
         var wModel = (WorkflowEventModel)model;
         this.Name = wModel.Name;
         this.Type = wModel.Type;
+        this.RunRepeatedly = wModel.RunRepeatedly;
+        this.DecisionOptionName = wModel.DecisionOptionName;
         this.Timer = wModel.Timer;
         this.BpmnElementId = wModel.BpmnElementId;
         this.CopyMixinsFrom(wModel);
         //WorkflowEventTaskModel.ApplyModel(this, wModel.Task);
     }
+
+    protected override string? PropertyValidation(PropertyInfo pi)
+    {
+        if (pi.Name == nameof(RunRepeatedly) && RunRepeatedly && Type != WorkflowEventType.BoundaryForkTimer)
+            RunRepeatedly = false;

```

Comment:
Changing fields in Property Validation is not so cool. Maybe move it to PreSaving?

Reply:
done and pushed